projpicker/web/web.py

```diff
@@ -0,0 +1,170 @@
+#!/bin/env python3
+import http.server
```

Comment:
Basic security checks?
* https://docs.python.org/3/library/http.server.html#module-http.server
* https://security.stackexchange.com/questions/226095/pythons-http-server-library-basic-security-checks

What about HTTPS and self-signed certificates?

Reply:
Just curious.. Is http.server the only higher-level module for creating a web server, I guess?

Reply:
Within the standard library, yes.

Reply:
I was wrong, there is also the [socketserver](https://docs.python.org/3/library/socketserver.html) and [ssl](https://docs.python.org/3/library/ssl.html) standard modules. Trying their implementation now.